File: BKFoodCourt/app/src/main/java/com/example/bkfoodcourt/Row.java
package com.example.bkfoodcourt;

public class Row {
    private String orderID;
    private String userID;

    public Row(String orderID, String userID) {
        this.orderID = orderID;
        this.userID = userID;
    }

    public String getOrderID() {
        return orderID;
    }

    public String getUserID() {
        return userID;
    }

    public void setOrderID(String orderID) {
        this.orderID = orderID;
    }

    public void setUserID(String userID) {
        this.userID = userID;
    }
}

File: BK_FoodCourt/app/src/main/java/com/example/bk_foodcourt/Row.kt
package com.example.bk_foodcourt

import android.widget.Button

class Row(s: String, s1: String) {
    private var orderID: String = ""
    private var userID: String = ""

    public fun Row(orderID: String, userID: String) {
        this.orderID = orderID
        this.userID = userID
    }
    public fun getOrderID(): String {
        return this.orderID
    }
    public fun getUserID(): String {
        return this.userID
    }
    public fun setOrderID(orderID: String) {
        this.orderID = orderID
    }
    public fun setUserID(userID: String) {
        this.userID = userID
    }
}